db/models/articles.models.js

```diff
@@ -1,5 +1,21 @@
 const db = require("../connection");
 
+exports.fetchAllArticles = () => {
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
+      COUNT (comment_id) AS comment_count
```

Comment:
It's worth having a look at the [CAST](https://www.w3schools.com/sql/func_sqlserver_cast.asp) function